test-outofproc/AddProductColumnTypes.cs

```diff
@@ -26,7 +26,7 @@ public static class AddProductColumnTypes
             var product = new ProductColumnTypes()
             {
                 ProductID = int.Parse(queryStrings["productId"], null),
-                Datetime = DateTime.UtcNow.Date,
+                Datetime = DateTime.ParseExact(DateTime.UtcNow.ToString(System.Globalization.CultureInfo.InvariantCulture), "yyyy-MM-ddTHH:mm:ss.fffZ", null),
```

Comment:
Out of curiosity have you tried using https://learn.microsoft.com/en-us/dotnet/api/system.data.sqltypes.sqldatetime?view=net-7.0?

Does something like this work?

```
DateTime = new SqlDateTime(DateTime.UtcNow).ToSqlString(),
```

Reply:
Didn't know about SqlDateTime, going to give it a try and see if it works. Thanks for the suggestion!